augur/export_v2.py

```diff
@@ -877,6 +887,9 @@ def register_parser(parent_subparsers):
                                  help="delimiters to accept when reading a metadata file. Only one delimiter will be inferred.")
     optional_inputs.add_argument('--metadata-id-columns', default=DEFAULT_ID_COLUMNS, nargs="+",
                                  help="names of possible metadata columns containing identifier information, ordered by priority. Only one ID column will be inferred.")
+    optional_inputs.add_argument('--metadata-columns', nargs="+",
+                                 help="Metadata columns to export in addition to columns provided by --color-by-metadata or --auspice-config. " +
```

Comment:
Deserves a changelog entry

Reply:
Added in 4bf28ec0d22e2560e6682681927d1fc581c6c647